### PR TITLE
Add bearerToken constructor to Authentication class

### PR DESCRIPTION
**Pull request by kevmoo**
Also added `authorizationHeaderValue()` function.
Also added `const` qualifier to `basic` constructor.

### CODE REVIEW

**Comment by kevmoo** [comment]
I'm still a bit confused about what to do w/ the changelog and the version in the pubspec @robbecker-wf  and @CaseyHillers

File: lib/src/common/util/auth.dart [hunk 1]
```diff
@@ -9,26 +11,65 @@ class Authentication {
...
+  // NEXT major version
   /// Creates an [Authentication] instance that uses the specified OAuth2 [token].
   const Authentication.withToken(this.token)
       : username = null,
-        password = null;
+        password = null,
+        bearerToken = null;
+
+  /// Creates an [Authentication] instance that uses the specified
+  /// [bearerToken].
```
Comment (CaseyHillers) [inline lib/src/common/util/auth.dart:25]: Is the line wrap necessary?

**Review by CaseyHillers — APPROVED** [review]

**Comment by CaseyHillers** [comment]
> I'm still a bit confused about what to do w/ the changelog and the version in the pubspec

Adding the related "semver: $change" label will have it get autogenerated. It uses the PR title as the changelog description

File: lib/src/common/util/auth.dart (hunk 1 above)
Comment (kevmoo) [reply lib/src/common/util/auth.dart:25]: To keep within 80 columns and keep the `.` – yes!

**Comment by robbecker-wf** [comment]
Released in version 9.17.0 https://github.com/SpinlockLabs/github.dart/releases/tag/9.17.0

**Comment by robbecker-wf** [comment]
The auto-release on PR merge isn't working for people other than myself due to permissions, so I triggered a release manually for now until I get a moment to fix it.

**Pull request merged**